docs/docs/installation/index.md

```diff
@@ -0,0 +1,44 @@
+# Install Mathesar
+
+This page contains instructions to install Mathesar on various platforms.
+
+## Guided installation using our install script
+- Use our install script to guide you install Mathesar via a series of prompts:
+    ```sh
+    bash <(curl -sfSL https://raw.githubusercontent.com/centerofci/mathesar/0.1.1/install.sh)
+    ```
+- The script sets up Mathesar using Docker Compose and configures the admin user [under the hood](./guided-install/under-the-hood.md).
+- !!! warning "Limitations"
+    This is a convenient way to install Mathesar. However, it requires `sudo` privileges (admin access), and sets up a limited set of configuration options. Use the [Docker Compose installation](./docker-compose/index.md) option to exert more control.
```

Comment:
The docker compose installation link is confusing here. I can see the 'Docker Compose' heading right below, and haven't built up enough context to avoid being confused when the link whisks me away somewhere else.

I suggest either removing the link, or just linking to the 'Docker Compose' Section on the main page here.